builders/treeFromInfo.nix

```diff
@@ -92,7 +92,7 @@ in {
     PATH    = "${coreutils}/bin:${findutils}/bin:${jq}/bin:${bash}/bin";
     args = ["-eu" "-o" "pipefail" "-c" ( ''
       mkdir -p "$out/node_modules";
-    '' + cmdFile.text )];
+    '' + "bash ${cmdFile}" )];
```

Comment:
Should we use 'source' instead of `bash`?

Reply:
Yeah that should be fine